rootfs/api/settings/production.py

```diff
@@ -256,6 +256,29 @@
 SLUGRUNNER_IMAGE = os.environ.get('SLUGRUNNER_IMAGE_NAME', 'quay.io/deisci/slugrunner:canary')  # noqa
 IMAGE_PULL_POLICY = os.environ.get('IMAGE_PULL_POLICY', "IfNotPresent")  # noqa
 
+# True, true, yes, y and more evaluate to True
+# False, false, no, n and more evaluate to False
+# https://docs.python.org/3/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool
+# see the above for all available options
+#
+# If a user deploys one build with a Procfile but then forgets to in the next one
+# then let that go through without scaling the missing process types down
+#
+# If the user has a Procfile in both deploys then processes are scaled up / down as per usual
+#
+# By default the process types are scaled down unless this setting is turned on
+DEIS_DEPLOY_PROCFILE_MISSING_REMOVE = strtobool(os.environ.get('DEIS_DEPLOY_PROCFILE_MISSING_REMOVE', 'true'))  # noqa
```

Comment:
`strtobool` appears to return a number instead of a boolean. Is this what we'd like?

```
><> python3
>>> bool('true')
True
>>> bool('false')
True
>>> from distutils.util import strtobool
>>> strtobool('false')
0
>>> strtobool('true')
1
```

Perhaps `bool(strtobool(...))` would be preferred here?